packages/catalog-model/src/entity/Entity.ts

```diff
@@ -112,6 +118,23 @@ export type EntityMeta = JsonObject & {
    */
   namespace?: string;
 
+  /**
+   * A nice display name of the entity, to be presented in user interfaces
```

Comment:
```suggestion
   * A display name of the entity, to be presented in user interfaces
```

Does not have to be nice right? :)